test/ariaqueryhandler.spec.ts

```diff
@@ -195,6 +195,22 @@ describeChromeOnly('AriaQueryHandler', () => {
       await page.waitForSelector('aria/[role="button"]');
     });
 
+    it('should persist query handler bindings across navigations', async () => {
+      const { page, server } = getTestState();
+
+      // Reset page but make sure that execution context ids start with 1.
```

Comment:
This comment confuses me. How do you assert the ids?

Reply:
I don't know how how the execution context ids are generated. But it seems like navigating within one origin results in incrementing ids while navigating across origins resets the ids to 1 again. That's why I added the two navigations there.
The previous test just checked by reloading the page which did not reset the ids and therefor did not uncover this edge case.